### PR TITLE
DMD-1318 Enter Pin

### PR DESCRIPTION
**Pull request by tomasz-ludek**

### CODE REVIEW

**Comment by HashEngineering** [comment]
@tomasz-ludek This needs to be merged with `redesign` after we get the Transaction Details PR merged.

File: common/src/main/java/org/dash/wallet/common/util/MonetarySpannable.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ private static CharSequence format(final MonetaryFormat format, final boolean si
         if (monetary == null)
             return "";
 
-        checkArgument(monetary.signum() >= 0 || signed);
+        //checkArgument(monetary.signum() >= 0 || signed);
```
Comment (HashEngineering) [inline common/src/main/java/org/dash/wallet/common/util/MonetarySpannable.java]: Why is this commented out?

File: wallet/AndroidManifest.xml [hunk 2]
```diff
@@ -233,6 +233,15 @@
...
 
+        <activity android:name="de.schildbach.wallet.ui.ReceiveActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/My.Theme.ChildActivity" />
+
+        <activity
+            android:name="de.schildbach.wallet.ui.TransactionResultActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/My.Theme.Fullscreen" />
+
```
Comment (HashEngineering) [inline wallet/AndroidManifest.xml]: Seems like something isn't working correctly with GitHub's diff.  This code was added in the Pay/Receive PR and it is already in the `redesign` branch.

Comment (HashEngineering) [reply wallet/AndroidManifest.xml]: Nevermind on this.

File: common/src/main/java/org/dash/wallet/common/util/MonetarySpannable.java (hunk 1 above)
Comment (tomasz-ludek) [reply common/src/main/java/org/dash/wallet/common/util/MonetarySpannable.java]: Hmm... It was commented out by @sambarboza , I don't know why.

Comment (HashEngineering) [reply common/src/main/java/org/dash/wallet/common/util/MonetarySpannable.java]: The checkArgument is checking for positive numbers, but we are now using this method to format negative numbers.

**Pull request merged**